package.json

```diff
@@ -14,7 +14,7 @@
     "doctoc": "doctoc --no-title --maxlevel 3 README.md CONTRIBUTING.md packages/*/README.md",
     "website:storybook": "lerna run storybook:build",
     "website:examples": "lerna run website:examples",
-    "lint": "eslint --ext .js,.ts,.tsx .",
+    "lint": "eslint --ext .js,.ts,.tsx . && lerna run lint",
```

Comment:
this is correct, but Create InstantSearch App lint should probably be removed, I have another stashed change for that

Reply:
https://github.com/algolia/instantsearch.js/pull/5217

Reply:
I approved your PR, I'll let you merge it then rebase this one so it passes.